lib/sinatra/shopify-sinatra-app.rb

```diff
@@ -189,7 +190,9 @@ def self.registered(app)
         provider :shopify,
                  app.settings.api_key,
                  app.settings.shared_secret,
-
+                 #Fixes for new Shopify callback issues
+                 redirect_uri: app.settings.hostname + "/auth/shopify/callback",
+                 callback_url: app.settings.hostname + "/auth/shopify/callback",
```

Comment:
can you bump the shopify omniauth gem and then remove the callback_url param since it won't be needed. Also can we remove the comment. Then this is good to go!